app/markets/ActivityTable.tsx

```diff
@@ -96,6 +127,9 @@ export const ActivityTable = ({ id }: { id: string }) => {
                     >
                       {shortenAddress(activity.creator.id)}
                     </a>
+                    {getIsAIAgent(activity.creator.id) && (
```

Comment:
why the get in the name? 

I think this would make it more clear:

```suggestion
                    {isAIAgent(activity.creator.id) && (
```